test/mod/test_testutil_net.py

```diff
@@ -0,0 +1,39 @@
+import os.path
+import pathlib
+import subprocess
+
+from osbuild.testutil import make_fake_tree
+from osbuild.testutil.net import http_serve_directory, https_serve_directory
+
+
+def test_http_serve_directory_smoke(tmp_path):
+    make_fake_tree(tmp_path, {
+        "file1": "file1 content",
+        "dir1/file2": "file2 content",
+    })
+    with http_serve_directory(tmp_path) as httpd:
+        output = subprocess.check_output(
+            ["curl", f"http://localhost:{httpd.server_port}/file1"])
+        assert output == b"file1 content"
+        output = subprocess.check_output(
+            ["curl", f"http://localhost:{httpd.server_port}/dir1/file2"])
+        assert output == b"file2 content"
+
+
+def test_http_ssl_serve_directory_smoke(tmp_path):
```

Comment:
For the function we are using `http` and `https` prefixes to diff ssl and non ssl version. I think the same convention can be used here instead of `ssl` keyword in function name.

Reply:
Thank you, updated.